viewing_party/party.py

```diff
@@ -1,23 +1,121 @@
-# ------------- WAVE 1 --------------------
+# ---------------------------------------------------- WAVE 1 ----------------------------------------------------
+#-----------------------------------------------------------------------------------------------------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+    
+    new_movie_info = {"title": title, "genre": genre, "rating": rating}
+    return new_movie_info
 
-# -----------------------------------------
-# ------------- WAVE 2 --------------------
-# -----------------------------------------
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
 
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
 
-# -----------------------------------------
-# ------------- WAVE 3 --------------------
-# -----------------------------------------
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)

```

Comment:
For this line, did we write a function that could do this for us? I know it's trivial here, but it helps to get in the habit so we can keep our code DRY. Also, why write a function if we won't use it? ✨ Great work on this function overall!

Reply:
fixed: will wait to push